lib/navigation/app_router.dart

```diff
@@ -51,6 +57,14 @@ class AppRouter {
             path: RoutePath.signIn.routePath,
             builder: (BuildContext context, GoRouterState state) =>
                 const SignInScreen(),
+            redirect: (BuildContext context, GoRouterState state) async {
```

Comment:
```suggestion
            redirect: (_, __) {
```

@hoangnguyen92dn we don't need `async` here. One more thing, we can omit the parameters if they are unused 😄

Reply:
@manh-t Good catch 👍